src/rules/percent-placeholder-pattern/index.js

```diff
@@ -0,0 +1,70 @@
+import _ from "lodash"
```

Comment:
Sure. The used variant is a rudiment from when some util functions were inside this file.